README.md

```diff
@@ -73,6 +73,16 @@ Below are the environment needed specifically for CCD domain.
    * DEFINITION_IMPORTER_PASSWORD: Password of the user on behalf of which definitions 
      will be imported to Definition Store, for automated test data preparation.
 
+Below are the environment needed specifically to Create Role Assignment data.
+* ROLE_ASSIGNMENT_HOST: Base URL of the Role Assignment APIs.
+* ROLE_ASSIGNMENT_USER_EMAIL: Email id of the user who can access role Assignment service.
+* ROLE_ASSIGNMENT_USER_PASSWORD: Password of the user who can access role Assignment service.
```

Comment:
considering the unit tests have a lot more env vars defined. I think here we need to define all MUST have environment variables here - can you check to see if we need to add more here

Reply:
Added the required variables, few ones like ROLE_ASSIGNMENT_SOLICITOR_USER,ROLE_ASSIGNMENT_SUPER_USER,role-assignment-staff1-user and their passwords are the dynamic variables in json files. So I haven't added them.